src/components/admonition/admonition.scss

```
@@ -0,0 +1,41 @@
+.admonition {
+    display: flex;
+    width: 100%;
+
+    &.danger {
+        background-color: rgb(var(--color-red-dark));
+    }
+    &.info {
+        background-color: rgb(var(--color-cyan-default));
+    }
+    &.note {
+        background-color: rgb(var(--color-gray-default));
+    }
+    &.tip {
+        background-color: rgb(var(--color-green-dark));
+    }
+    &.warning {
+        background-color: rgb(var(--color-orange-dark));
+    }
```

I'm thinking we might want to allow overriding the default colours by doing something like:

```suggestion
    &.danger {
        background-color: var(--admonition-danger-color, rgb(var(--color-red-dark)));
    }
    &.info {
        background-color: var(--admonition-info-color, rgb(var(--color-cyan-default)));
    }
    &.note {
        background-color: var(--admonition-note-color, rgb(var(--color-gray-default)));
    }
    &.tip {
        background-color: var(--admonition-tip-color, rgb(var(--color-green-dark)));
    }
    &.warning {
        background-color: var(--admonition-warning-color, rgb(var(--color-orange-dark)));
    }
```

What do you think @Kiarokh?

---

It could be good. I agree

---

This is also something we can add later though. Better to make it simpler now, and add features _later_.

---

I'll add this as a fixup now. For me it's easier to think about these features already when the component is being prepared and the design is being massaged. :)